cookbooks/aws-parallelcluster-environment/resources/nfs/partial/_configure.rb

```diff
@@ -24,6 +24,8 @@
     service node['nfs']['service']['server'] do
       action %i(restart enable)
       supports restart: true
+      retries 5
+      retry_delay 10
```

Comment:
We should have the same retry in every other service resource in our cookbook, if missing.